site/admin/theatres/edit.php

```diff
@@ -0,0 +1,151 @@
+<?php
+include_once('init.php');
+PrintHeader('Edit Theatre', 'Edit Theatre');
+
+
+echo"<div class='ContactForm'>
+	<form action='' method='get'>
+	Which theatre do you want to edit?
+	<br /><br />";
+@TextBox('EditTheatre');
+echo"<br /><br />
+	<input type='submit' name='EditTheatreSubmitted' />
+	</form>
+	</div>";
+
+if(isset($_REQUEST['EditTheatreSubmitted'])){
+	if(!@$_REQUEST['EditTheatre']){
+		$FormErrors['EditTheatre'] = "Please enter a theatre to edit.";
+	}
+	if(sizeof($FormErrors) == 0){
+		$TheatreInfo = GetTheatre($_REQUEST['EditTheatre']);
+		//echo"".$TheatreInfo['Email']."";
+		//var_dump($TheatreInfo);
+		foreach($TheatreInfo as $Row){
+			echo"<div class='ContactForm'>
+				<h3>Edit the theatre information here:</h3>
+				<br />
+				<form action='' method='post'>
+				Theatre Id:
+			<br /><br />";
+			@ReadOnlyTextBox($Row['TheatreId'], 'TheatreId');
```

Comment:
I'm curious what the `@` sign is for here. If this function is throwing off errors, it might be worth figuring out why and fixing them.